.github/workflows/lte-integ-test-magma-deb.yml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: LTE integ test magma-deb
+
+on:
+  workflow_dispatch: null
+  workflow_run:
+    workflows:
+      - build-all
+    branches:
+      - master
+    types:
+      - completed
+
+jobs:
+  lte-integ-test-magma-deb:
+    if: github.repository_owner == 'magma' || github.event_name == 'workflow_dispatch'
+    runs-on: macos-12
+    env:
+      SHA: ${{ github.event.workflow_run.head_commit.id || github.sha }}
+    steps:
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+        with:
+          ref: ${{ env.SHA }}
+      - name: Cache magma-deb-box
+        uses: actions/cache@0865c47f36e68161719c5b124609996bb5c40129 # pin@v3
+        with:
+          path: ~/.vagrant.d/boxes/magmacore-VAGRANTSLASH-magma_deb
+          key: vagrant-box-magma-deb-v1.2.20220801
```

Comment:
what does the suffix `v1.2.20220801` mean in this case? Does it make more sense to code the used image into the key?
```
magma_deb.vm.box = "ubuntu/focal64"
magma_deb.vm.box_version = "20220804.0.0"
```
-> e.g., `focal64_20220804.0.0`

Reply:
I only followed the `lte-integ-test.yml` and ` lte-integ-test-bazel.yml`. Do you think we need more then one deb.vm version in the cache?

Reply:
No, we only need one - but in `lte-integ-test.yml` the cache name is based on the magma (_dev) base image. I.e., there is imho no reason to use this name here - there is just no connection (magma_deb is not based on the magma(_dev) base image).

Reply:
Ok, make sense.

Reply:
Is solved in a offline session